src/clj/athenaeum/domain/user.clj

```diff
@@ -0,0 +1,17 @@
+(ns athenaeum.domain.user
+  (:require [next.jdbc.sql :as sql]))
+
+(defn fetch-by-google-id
+  [c google-id]
+  (first (sql/find-by-keys c
+                           :users
+                           {:google_id google-id})))
+
+(defn create
+  [c user]
+  (sql/insert! c
+               :users
+               {:google_id (:sub user)
```

Comment:
It's a little strange that this function expects the google ID to be under the `:sub` key. We can do away with that, and look up the google ID from the `:google-id` key or something, which seems more appropriate.

Reply:
Obviously the callers will have to be changed to pass in the map with the appropriate keys.